static/redfish/v1/schema/OemComputerSystem_v1.xml

```diff
@@ -95,6 +95,11 @@
                   <Annotation Term="OData.Description" String="An indication of PEL topology information is saves."/>
                   <Annotation Term="OData.LongDescription" String="This property when set to 'true' saves the topology information. The information is saved in the form of a PEL(Error Log)."/>
                 </Property>
+                <Property Name="SafeMode" Type="OemServiceRoot.IBM">
```

Comment:
Type would need to be `Edm.Boolean` to satisfy Redfish Service Validator.
The correct one is on 1060 branch
https://github.com/ibm-openbmc/bmcweb/blob/e01a12ccef4be156f0ac3383e267b77b1cf29dda/static/redfish/v1/schema/OemComputerSystem_v1.xml#L88